docs/standard/base-types/backtracking-in-regular-expressions.md

```diff
@@ -112,8 +112,8 @@ Backtracking occurs when a regular expression pattern contains optional [quantif
  [!code-csharp[System.Text.RegularExpressions.Regex.ctor#1](../../../samples/snippets/csharp/VS_Snippets_CLR_System/system.text.regularexpressions.regex.ctor/cs/ctor1.cs#1)]
  [!code-vb[System.Text.RegularExpressions.Regex.ctor#1](../../../samples/snippets/visualbasic/VS_Snippets_CLR_System/system.text.regularexpressions.regex.ctor/vb/ctor1.vb#1)]  
 
-### Nonbacktracking Subexpression  
- The `(?>` *subexpression*`)` language element suppresses backtracking in a subexpression. It is useful for preventing the performance problems associated with failed matches.  
+### Atomic groups
+ The `(?>` *subexpression*`)` language element suppresses backtracking in a subexpression. This can be useful for preventing the performance problems associated with failed matches.  
```

Comment:
Not part of your change, but I think the text as it was written is misleading.  Saying "suppresses backtracking in a subexpression" makes it sound (at least to me) like there won't be any backtracking for any constructs in the subexpression... and that's not true.  Atomic groups prevent the subexpression from giving up anything that was matched to anything that comes after it, meaning it prevents backtracking _into_ the subexpression.  For example, if you have `(?>\w*\d*)1`, there may be backtracking as part of the `\d` failing to match, but once the `\w*\d*` has successfully matched and we exit the atomic group, if the `1` fails to match, we won't then re-enter the atomic group to backtrack into the `\d*` or the `\w*`.  In effect, the atomic group puts up a barrier around it that causes any attempts at backtracking to skip over it.
```suggestion
 The `(?>` *subexpression*`)` language element suppresses backtracking into a subexpression. This can be useful for preventing the performance problems associated with failed matches.  
```

Reply:
changed to
```
### Atomic groups
 The `(?>` *subexpression*`)` language element suppresses backtracking into the subexpression. Once it has successfully matched, it will not give up any part of its match to subsequent backtracking. For example, in the pattern `(?>\w*\d*)1`, if the `1` cannot be matched, the `\d*` will not give up any of its match even if that means it would allow the `1` to successfully match. Atomic groups can help prevent the performance problems associated with failed matches.
```